lib/gamora.ex

```diff
@@ -234,6 +235,13 @@ defmodule Gamora do
     end
   end
 
+  defp with_max_age_param(opts, conn) do
+    case conn.params["max_age"] || option(conn, :default_max_age) do
```

Comment:
IMO we Should not add logic in a case condition, so I will move `conn.params["max_age"] || option(conn, :default_max_age)`. to a `fetch_max_age(conn)`. priv function, or do it in a local variable above the case.

Reply:
Improved, please take another look.